tests/unittests/OperatorTest.cpp

```diff
@@ -1358,6 +1358,52 @@ TEST_P(Operator, ChannelShuffle) {
     EXPECT_FLOAT_EQ(results.at({0, i, 0, 0}), expected[i]);
 }
 
+TEST_P(Operator, Squeeze) {
+  auto *inputs = mod_.createVariable(ElemKind::FloatTy, {1, 2, 1, 5}, "inputs");
```

Comment:
You're creating this as `FloatTy`, but below your vector of comparison values is `int`.